src/cpp/lpnamer/model/Problem.h

```diff
@@ -103,6 +103,7 @@ class Problem: public SolverAbstract
   void set_simplex_iter(int iter) override { solver_abstract_->set_simplex_iter(iter);}
   void write_basis(const std::filesystem::path &filename) override { solver_abstract_->write_basis(filename); }
   void read_basis(const std::filesystem::path &filename) override { solver_abstract_->read_basis(filename); }
+  void SetBasis(std::vector<int> rstatus, std::vector<int> cstatus) override { solver_abstract_->SetBasis(rstatus, cstatus); }
```

Comment:
Why do you chose a different formatting in the function name compared to the other functions of the class ? (Because the input data type is different ?)

Reply:
That's a difficult point.
On one hand we should keep formatting homogeneous in files, on another hand is we don't take the habits to use our chosen format we will never do.